packages/components/src/Dropzone/sub-components/SelectedFileMessage/SelectedFileMessage.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react"
+import { Button } from "@kaizen/button"
+import { Icon } from "@kaizen/component-library"
+import selectedFileSvgIcon from "@kaizen/component-library/icons/file.icon.svg"
+import acceptedFileSvgIcon from "@kaizen/component-library/icons/success.icon.svg"
+import { Paragraph } from "@kaizen/typography"
+import { trunicateFilename } from "../../utils"
+import { Hyperlink } from "../Hyperlink/Hyperlink"
+
+interface SelectedFileMessageProps {
+  fileName: string
+  handleOpenDialog: () => void
+}
+
+export const AcceptedFileIcon = (): JSX.Element => (
+  <span className="inline-block mb-8 text-green-500">
+    <Icon
+      height={20}
+      width={20}
+      icon={acceptedFileSvgIcon}
+      title="Selected File"
+      role="presentation"
+      inheritSize={true}
+    />
+  </span>
+)
+
+export const SelectedFileMessage = ({ fileName, handleOpenDialog }: SelectedFileMessageProps): JSX.Element => {
+  const maxCharacterLimit = 50
+  let selectedFileName = fileName
+
+  if (fileName?.length > maxCharacterLimit) {
+    selectedFileName = trunicateFilename(fileName, maxCharacterLimit)
+  }
+
+  return (
+    <div className="flex flex-col items-center justify-center">
+      <span className="mb-8">
```

Comment:
Similar to the icons, it might be simpler to lift this wrapper to `Dropzone.tsx` and have all messages live inside it:

```
<div className="mb-8">
    <SelectedFileMessage />
    <DefaultMessage />
</div>
```